src/WooCommerceShippingPackages.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\ShopStandards\WooCommerceShippingPackages.
+ */
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Shipping related settings and actions.
+ */
+class WooCommerceShippingPackages {
+
+  /**
+   * Initialize the needed functions and hooks.
+   */
+  public static function init() {
+    // Show multiple shipping methods separately in order emails.
+    add_filter('woocommerce_get_order_item_totals', __CLASS__ . '::woocommerce_get_order_item_totals', 10, 2);
+  }
+
+  /**
+   * Calculates the shipping costs including tax.
+   *
+   * @param \WC_Order_Item_Shipping $shipping_method
+   *   A shipping item object derived from a WC_Order.
+   *
+   * @return string
+   *   The shipping costs including taxes, currency sign, and HTML line-breaks,
+   *   for output in the HTML order emails.
+   */
+  public static function getShippingCostsIncludingTaxes(\WC_Order_Item_Shipping $shipping_method): string {
+
+    if ($shipping_price) {
+      $currency = ' ' . get_woocommerce_currency_symbol();
+      $shipping_price = floatval($shipping_method->get_total());
+      $shipping_price_tax = floatval($shipping_method->get_total_tax());
+      $shipping_price_total = $shipping_price + $shipping_price_tax;
+      $shipping_price_total_with_symbol = ' - ' . $shipping_price_total . $currency . '<br />';
+
+      return $shipping_price_total_with_symbol;
+    }
+
+    return '<br />';
+  }
+
+  /**
+   * Separates the shipping methods in new lines in emails.
+   *
+   * @param array $total_rows
+   *   Total rows of woocommerce order items.
+   * @param \WC_Order $order
+   *   A Wc_Order object.
+   *
+   * @return array
+   *   The shipping details including shipping method name, shipping costs,
+   *   taxes and currency sign.
```

Comment:
```suggestion
   * @implements woocommerce_get_order_item_totals
```

Reply:
As mentioned here: https://github.com/netzstrategen/wordpress-shop-standards/pull/218#discussion_r1005524253

if you find a more suitable initial description for the method, you can also change it. IMO, it explains quite good what we are trying to accomplish.